src/test/resources/features/ContactDetailsAndRegistration.feature

```diff
@@ -175,107 +175,82 @@ Feature: Contact details for the filing member
     When I click on change link for Second Contact Telephone number
-    When I enter Second Contact Input as 71235643
+    And I enter Second Contact Input as 71235643
     And I click on Continue button
     When I click on change link for Address
-    When I enter Address Line 1 as Address Change
-    And I click on Continue button
-    And I should see row 1 value Contact Name Change
-    And I should see row 2 value emailchange@test.com
-    And I should see row 4 value 1234555
-    And I should see row 6 value Second Contact Name Change
-    And I should see row 7 value secondContactchange@email.com
-    And I should see row 8 value Yes
-    And I should see row 9 value 71235643
-    And I should see row 10 value Address Change
+    And I enter Address Line 1 as Address Change
+    And I click on Continue button
+    Then I should see details as below:
+      | KEY                             | VALUE                         |
+      | Contact name                    | Contact Name Change           |
+      | Email address                   | emailchange@test.com          |
+      | Telephone number                | 1234555                       |
+      | Second contact name             | Second Contact Name Change    |
+      | Second contact email address    | secondContactchange@email.com |
+      | Can we contact by telephone?    | Yes                           |
+      | Second contact telephone number | 71235643                      |
+      | Address                         | Address Change                |
     And I click on Continue button
     When I click Check your answers link
     Then I should be on Review answers page
-    And I should see row 1 key Name
-    And I should see row 2 key Address
-    And I should see row 3 key Contact name
-    And I should see row 4 key Email address
-    And I should see row 5 key Can we contact by telephone?
-    And I should see row 6 key Telephone number
-    And I should see row 1 value Test
-    And I should see row 2 value Address Line 1
-    And I should see row 2 value City
-    And I should see row 2 value EH5 5WY
-    And I should see row 2 value United Kingdom
-    And I should see row 3 value UPE Test
-    And I should see row 4 value testupe@email.com
-    And I should see row 5 value Yes
-    And I should see row 6 value 123456
-    And I should see row 7 key Is there a nominated filing member
-    And I should see row 7 value No
-    And I should see row 8 key Where are the entities in your group located?
-    And I should see row 9 key Group’s consolidated accounting period
-    And I should see row 10 key Start date
-    And I should see row 11 key End date
-    And I should see row 8 value In the UK and other countries
-    And I should see row 10 value 15 January 2024
-    And I should see row 11 value 15 January 2025
-    And I should see row 12 key Contact name
-    And I should see row 13 key Email address
-    And I should see row 14 key Can we contact by telephone?
```

Comment:
Do not delete these validation steps as they are not included in data table.

Reply:
remove key steps here, but covered values steps after data table